src/components/Content/ContentPanel.tsx

```diff
@@ -0,0 +1,75 @@
+import format from 'date-fns/format/index.js';
+import { injectState } from 'freactal';
+import { css } from 'glamor';
+import { groupBy, upperCase } from 'lodash';
+import React from 'react';
+import { compose } from 'recompose';
+import { Grid } from 'semantic-ui-react';
+
+import { DATE_KEYS } from 'common/injectGlobals';
+import ContentPanelView from './ContentPanelView';
+
+export const getFieldContent = (row, data) => {
+  return DATE_KEYS.indexOf(row.key) >= 0
```

Comment:
while technically fine ... my preference is to discourage ternary statements like this when they start containing `else if` logic. Reason being that they are supposed to be a useful shorthand but at this point you may as well just right the full if/else if/else block. Not sure what others position on this is so up to you, I always favour clarity and readability over small gains in compactness.

Reply:
that sounds reasonable to me. i think i was just extending what was already there but you're right, it does get needlessly confusing after the second if